frontend/src/components/common/table/table/styles.module.scss

```
@@ -0,0 +1,47 @@
+@import '../../../../assets/css/variables.scss';
```

And styles file should be on the same level as a component which it relates to (in this case on same placement as `table.tsx`)
```suggestion
@import 'assets/css/variables.scss';
```